src/BugsnagLogger.php

```diff
@@ -26,6 +26,27 @@ class BugsnagLogger extends AbstractLogger
      */
     protected $notifyLevel = LogLevel::NOTICE;
 
+    /**
+     * The level for the current log record.
+     *
+     * @var string|null
+     */
+    protected $level;
```

Comment:
The logger is intended just for processing the log message, we shouldn't be storing state of a particular message here.

Reply:
I think it would be nice to not store the state, but 1 - I did not see a nice way to do it 2 - As all processing start with log() method, the state will always be reseted.

However, if you like, I can move the processing to a stateless class OR simply unset the vars after the process in log().